deploy/managed-upgrade-operator-config/4.7/config.yaml

```diff
@@ -0,0 +1,6 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+  - key: hive.openshift.io/version-major-minor
+    operator: In
+    values: ["4.7"]
```

Comment:
Let's move this config up to the parent folder with a NotIn 4.5 or 4.6. We want a "default" config that applies to the current latest version of OSD, plus things like 4.8 nightlies

Reply:
Thanks @cblecker ! Added the changes as requested